tpl/template_embedded.go

```diff
@@ -233,7 +233,7 @@ func (t *GoHTMLTemplate) EmbedTemplates() {
 {{ end }}{{ end }}
 
 <!-- Output all taxonomies as schema.org keywords -->
-<meta itemprop="keywords" content="{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}" />
+<meta itemprop="keywords" content="{{ if .IsPage}}{{ range $index, $tag := .Params.tags }}{{ $tag }},{{ end }}{{ else }}{{ range $plural, $terms := .Site.Taxonomies }}{{ range $term, $val := $terms }}{{ printf "%s," $term }}{{ end }}{{ end }}{{ end }}" />
```

Comment:
Not to poop and swoop. My suggested resolution is to do this:

```
<meta itemprop="keywords" content="{{ .Keywords }}">
```

If if you're trying to do this at the page level _only_, you would do something like this:

```html
<meta name="keywords" content="{{ if .Keywords }}{{ delimit .Keywords ", " }}{{ else if .IsPage }}{{ delimit $.Params.tags ", " }}{{ else }}{{ range $term, $val := .Site.Taxonomies.categories }}{{ lower (humanize $term) }}, {{ end }}{{ end }}">
```

The only deficiency in the above page-level code AFAICT is there is an extra comma at the end of the list of commas. Annoying for humans, parsers won't care.